manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/AbstractWorkerTest.java

```diff
@@ -0,0 +1,64 @@
+package com.redhat.service.bridge.manager.workers.resources;
+
+import java.time.ZonedDateTime;
+
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.redhat.service.bridge.infra.models.dto.ManagedResourceStatus;
+import com.redhat.service.bridge.manager.RhoasService;
+import com.redhat.service.bridge.manager.TestConstants;
+import com.redhat.service.bridge.manager.models.Bridge;
+import com.redhat.service.bridge.manager.models.Work;
+import com.redhat.service.bridge.manager.workers.WorkManager;
+import com.redhat.service.bridge.test.resource.PostgresResource;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.verify;
+
+@QuarkusTest
+@ExtendWith(MockitoExtension.class)
+@QuarkusTestResource(PostgresResource.class)
+public class AbstractWorkerTest {
+
+    private static final String RESOURCE_ID = "123";
+
+    @InjectMock
+    WorkManager workManager;
+
+    @InjectMock
+    @SuppressWarnings("unused")
+    //Needed to set up RHOAS for tests
+    RhoasService rhoasService;
+
+    @Inject
+    BridgeWorker worker;
+
+    @Test
+    @Transactional
+    void workIsCompletedWhenResourceIsNotFound() {
+        Bridge bridge = new Bridge(TestConstants.DEFAULT_BRIDGE_NAME);
+        bridge.setCustomerId(TestConstants.DEFAULT_CUSTOMER_ID);
+        bridge.setStatus(ManagedResourceStatus.ACCEPTED);
+        bridge.setSubmittedAt(ZonedDateTime.now());
+        bridge.setId(RESOURCE_ID);
```

Comment:
Is this code needed? The `bridge` instance is used below just to retrieve submitted time - that can be set directly

Reply:
Yes, the call [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/475/files/6429a6ec9d4d9f2717eaf2e30ab188c6ac44c70c#diff-95c3db7d40e5c0e6398a83f2285f23a90e93db0975bafe77eebe504afdad8109R59) needs other properties set in order for it to function correctly.

Reply:
```
    @Test
    @Transactional
    void workIsCompletedWhenResourceIsNotFound() {
        Work work = new Work();
        work.setType(Bridge.class.getName());
        work.setSubmittedAt(ZonedDateTime.now());
        work.setManagedResourceId(RESOURCE_ID);

        assertThatThrownBy(() -> worker.handleWork(work)).isInstanceOf(IllegalStateException.class);

        verify(workManager).complete(work);
    }
```

Reply:
Ha, erm, no not really.. I am clearly getting my PRs confused. Removed surplus setup.

It might well have been used before I [updated the test](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/475#issuecomment-1070876282).